librz/core/cil.c

```diff
@@ -578,6 +582,9 @@ RZ_IPI void rz_core_analysis_il_vm_status(RzCore *core, const char *var_name, Rz
 			case RZ_IL_TYPE_PURE_BOOL:
 				rzil_print_register_bool(val->data.b->b, &p);
 				break;
+			case RZ_IL_TYPE_PURE_FLOAT:
+				// TODO : print register float ?
```

Comment:
Same, please add basic printing here

Reply:
Supporting float literal as input in the command line interface ? then it could accept non-ut64 value. Before that I will add some basic printing here.